### PR TITLE
Update German localization

### LINKED ISSUE #188: IMPROVE - German localisation
## Describe the feature you want to improve
Update the missing strings in the Localizable file

### PR DESCRIPTION
**Pull request by Drag0ndust**
# What it Does
* Closes #188 
* Describe what your change does

# How I Tested
* Checked all screens for missing strings

### CODE REVIEW

File: Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme [hunk 1]
```diff
@@ -57,6 +57,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
+      region = "DE"
       launchStyle = "0"
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme]: Remove these
These are by default changing the language / region (when they should just stay as system defaults)

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
One little change

File: Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme (hunk 1 above)
Comment (Drag0ndust) [reply Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme]: sorry, added this for testing and accidental committed it. I will revert it

Comment (Drag0ndust) [reply Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme]: fixed it

**Review by mikaelacaron — APPROVED** [review]
Thanks for the update!

**Pull request merged**